src/appmixer/openai/core/createFile/createFile.js

```
@@ -84,17 +87,16 @@
-                    headers: req.headers
+                    headers: {
+                        ...req.headers,
+                        Authorization: req.Authorization ? 'Bearer ***' : req.Authorization
+                    }
                 },
                 response: err.response ? {
-                    data: err.response.data,
-                    status: err.response.status,
-                    statusText: err.response.statusText,
-                    headers: err.response.headers
+                    message: err.response.data.error.message,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)